workshop/exercise-4/README.md

```diff
@@ -17,19 +34,20 @@ Because Knative is built on top of Kubernetes, you can use kubectl along with a
     apiVersion: serving.knative.dev/v1alpha1
     kind: Service
     metadata:
-      name: fib-knative
-      namespace: default
+    name: fib-knative
```

Comment:
The indentation in this file seems off. E.g. name and namespace should be under `metadata` but now they're at the same top-level

Reply:
thanks - not sure how that happened.